src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views.test.tsx

```diff
@@ -12,31 +12,29 @@ import { ReactWrapper } from 'enzyme';
 import { mountWithIntl } from '@kbn/test-jest-helpers';
 import { EuiButton } from '@elastic/eui';
 
-import { ServicesProvider, SharedUXServices } from '../../../services';
-import { servicesFactory } from '../../../services/mocks';
+import {
+  SharedUxServicesProvider,
+  SharedUxServices,
+  mockServicesFactory,
+} from '@kbn/shared-ux-services';
```

Comment:
Do we need to import from package? Would relative path not work here?

Reply:
AFAIK you have to import from dependent packages.  It would be like importing from `@kbn/i18in` relatively.  The BAZEL.build and tsconfig would also have to list those source files as belonging to this package... I don't see how we'd make this relative.

@spalger Let me know if I'm wrong here.

Reply:
Yeah, the packages should be thought of as node modules and are pulled in from the node_modules directory. Relative imports should never be used to access them they should only be imported using their single module id